app/controllers/obituaries/covid_controller.rb

```diff
@@ -0,0 +1,4 @@
+class Obituaries::CovidController < ApplicationController
```

Comment:
Fun fact! Couldn't call this `Covid-19Controller` because rails was like "Warning: That Punctuation is Going to be a Problem" (not really, but basically)

Reply:
Interesting, do you think that is because of the hyphen?